recipe-server/normandy/recipes/tests/api/v2/test_api.py

```diff
@@ -195,10 +195,36 @@ def test_it_can_create_disabled_recipes(self, api_client):
             assert recipes.count() == 1
 
         def test_creation_when_action_does_not_exist(self, api_client):
-            res = api_client.post('/api/v2/recipe/', {'name': 'Test Recipe',
-                                                      'action_id': 1234,
-                                                      'arguments': '{}'})
+            res = api_client.post('/api/v2/recipe/', {
+                'name': 'Test Recipe',
+                'action_id': 1234,
+                'arguments': '{}',
+            })
+            assert res.status_code == 400
+            assert res.json()['action_id']
```

Comment:
I didn't want to make an assertion of the **value** because it's not our code really. It's Django REST Framework's wording.

Reply:
DRF exposes these as constants, so the comparison could be something like:

```python
assert res.json()['action_id'] == serializers.SlugRelatedField.default_error_message['does_not_exist'].format(pk_value=1234)
```

This is kind of wordy though. I think it is worth making sure that we got the right error out of this *somehow* though. This could be failing because `action_id` is an invalid key for all we know.